backend/src/test/java/com/yagubogu/stat/service/StatServiceTest.java

```diff
@@ -185,4 +187,30 @@ void findLuckyStadium_noCheckInCounts() {
         // then
         assertThat(luckyStadium.shortName()).isNull();
     }
+
+    @DisplayName("평균 득, 실, 실책, 안타, 피안타 조회한다")
```

Comment:
C: 테스트 잘 작성하셨네요 👍
이 부분 아래와 같이 고치면 더 자연스러운 것 같아요 :)
```suggestion
    @DisplayName("평균 득, 실, 실책, 안타, 피안타를 조회한다")
```